chilli.yml

```diff
@@ -149,74 +138,56 @@
         - coova-chilli
         - lua-cjson
         - rpcd-mod-iwinfo
-    # packages used in the nossl flavour
-    nnxx_nossl_packages:
-        - luci
+    content_filtering:
+        - privoxy
+        - e2guardian
     olsrd2_packages:
         - oonf-olsrd2
-    olsrd1_packages:
-        - olsrd
-        - olsrd-mod-nameservice
-        - olsrd-mod-arprefresh
-        - olsrd-mod-dyn-gw
-        - olsrd-mod-txtinfo
-        - olsrd-mod-jsoninfo
-        - luci-lib-json
-        - luci-app-olsr
     # packages used in flavours
-    full_olsrd2_packages: "{{ openwisp2fw_default_packages + nnxx_full_packages + olsrd2_packages + nnxx_chilli }}"
-    full_olsrd1_packages: "{{ openwisp2fw_default_packages + nnxx_full_packages + olsrd1_packages + olsrd2_packages + nnxx_chilli }}"
-    nossl_olsrd1_packages: "{{ openwisp2fw_default_packages + nnxx_nossl_packages + olsrd1_packages + olsrd2_packages + nnxx_chilli }}"
+    base_packages: "{{ openwisp2fw_default_packages + nnxx_full_packages + olsrd2_packages }}"
+    hotspot_packages: "{{ openwisp2fw_default_packages + nnxx_full_packages + olsrd2_packages + nnxx_chilli + content_filtering }}"
     # definition of available flavours
     openwisp2fw_image_flavours:
-        full_olsrd2:
+        base:
           ar71xx:
-            packages: "{{ full_olsrd2_packages }}"
+            packages: "{{ base_packages }}"
           x86:
-            packages: "{{ full_olsrd2_packages }}"
+            packages: "{{ base_packages }}"
           brcm2708:
-            packages: "{{ full_olsrd2_packages }}"
+            packages: "{{ base_packages }}"
           sunxi:
-            packages: "{{ full_olsrd2_packages + ['-kmod-eeprom-sunxi'] }}"
+            packages: "{{ base_packages + ['-kmod-eeprom-sunxi'] }}"
           ipq806x:
-            packages: "{{ full_olsrd2_packages }}"
-        full_olsrd1:
-          ar71xx:
-            packages: "{{ full_olsrd1_packages }}"
-          x86:
-            packages: "{{ full_olsrd1_packages }}"
-          ramips:
-            packages: "{{ full_olsrd1_packages }}"
-        nossl_olsrd1:
+            packages: "{{ base_packages }}"
+        hotspot:
           ar71xx:
-            packages: "{{ nossl_olsrd1_packages }}"
+            packages: "{{ hotspot_packages }}"
           x86:
-            packages: "{{ nossl_olsrd1_packages }}"
+            packages: "{{ hotspot_packages }}"
     # definition of organizations
     default_openwisp2_url: https://openwisp.nnxx.ninux.org
     openwisp2fw_organizations:
-        # - name: defaults
-        #   flavours:
-        #     - full_olsrd2
-        #     - full_olsrd1
-        #     - nossl_olsrd1
-        #   openwisp:
-        #     url: "{{ default_openwisp2_url }}"
-        #     shared_secret: Secretn1nuxB4s
-        #     unmanaged: "{{ openwisp2fw_default_unmanaged }}"
+        - name: defaults
```

Comment:
che organizzazione è questa? Mi pare uguale a `basilicata`, se così fosse non serve duplicarla giusto?

Reply:
non ha le customizzazioni che sono presenti in basilicata, sia nel banner che nella configurazione delle reti wifi

Reply:
lo usa qualcuno? Calcola che allunga il tempo di compilazione, se non lo usa nessuno leviamo anche questo.